### DISCUSSION TITLE
Change button background color

### DISCUSSION THREAD

**Discussion by cawecoy** [filamentphp/filament#10566] [Help] [answered]
Hi! How can I change the button main background color and its background color on hover? 

I've tried the code bellow, but it doesn't work:

```
<x-filament::button class="bg-custom-500 hover:bg-custom-600">
    Button
</x-filament::button>
```

**Comment by mahmudz** [comment, marked as answer]
@cawecoy You wil need the register the classes in your tailwind.config.js file

**Reply by cawecoy** [reply]
You are right. It looks like Filament doesn't have `.bg-custom-600` and `.hover\:bg-custom-500`, it just has `.bg-custom-500` and `.hover\:bg-custom-600`. Thanks!
